app/controllers/auth_controller.py

```diff
@@ -0,0 +1,17 @@
+from typing import Tuple
+from flask import request, Blueprint
+from app.models.user import User
+from app.services import auth_service
+
+auth_routes = Blueprint('auth_routes', __name__)
+
+
+@auth_routes.route('/signup', methods=['POST'])
+def sign_up() -> Tuple[str, int]:
+    if request.method == 'POST':
+        full_name, username, email, password = request.form.values()
+
```

Comment:
Validation?

Reply:
same as above we can add in a regex later if we want

Reply:
We need to use regex because we have to validate the emails especially within our eventual test cases.

Reply:
updated